### PR TITLE
added-loading-file

### PR DESCRIPTION
**Pull request by murtazshaikh**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```
Comment (syedabdullahali) [inline index.html:1]: ! DOCTYPE html  .What work of it

**Review by syedabdullahali — COMMENTED** [review]
great job

File: style.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+section{
+    min-height: 100vh;
+    background: #000;
+
```
Comment (mujeeb100) [inline style.css:9]: please remove the space in css file there are lots of space are there in your css file...

File: style.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    border-radius: 50%;
+    background: #00ff0a;
+    box-shadow: 0 0 10px #00ff0a,
+    0 0 20px #00ff0a,
+    0 0 40px #00ff0a,
+    0 0 60px #00ff0a,
+    0 0 80px #00ff0a,
+    0 0 100px #00ff0a;
+    animation: animate 2s linear infinite;
+    animation-delay: calc(0.1s * var(--i));
```
Comment (syedabdullahali) [inline style.css:56]: Using Custom variable great idea

**Review by syedabdullahali — APPROVED** [review]

**Review by mmusk — APPROVED** [review]

**Review by OSAMA478 — APPROVED** [review]
looks good

File: index.html (hunk 1 above)
Comment (murtazshaikh) [reply index.html:1]: It specifies the version of an html document to browser.

File: style.css (hunk 2 above)
Comment (murtazshaikh) [reply style.css:9]: okk

**Pull request merged**